src/tests/unit_tests/test_document_converter.py

```diff
@@ -287,7 +297,7 @@ def test_convert_firestore_document_with_filters(
                 "data": {
                     "page_content": "value",
                     "metadata_field": DocumentReference(
-                        *["abc", "xyz"], client=pytest.client
```

Comment:
if we are removing all usage of `pytest.client` the config file needs to be removed then (?)

Reply:
Yes